src/sys/wait.rs

```diff
@@ -25,13 +27,24 @@ libc_bitflags!(
           target_os = "android"))]
 libc_bitflags!(
```

Comment:
I'd like to see a doccomment for `WaitPidFlag` here since you've already documented (nearly) every item.